### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by xcvb85** [HHS81/do328#49]
Here we can ask questions. Maybe someone knows the answer.

Unanswered Questions:
3.) Q: What is shown on MFD page "MAIN 2/2"?

Answered Questions (answer not sure):
1.) Q: How can I change the PFD styles (V-Bar and Cross-Bar Format)?
1.) A: I can imagine that for security reasons this must be done by the mechanics on the ground and that  there is no way to change that by pressing a simple button. We could change the style depending on the selected livery but this is of course very low priority.
4.) Q: What is the content of "REF DATA"->"CRUISE"?
4.) A: VMO and VNE

Answered Questions (answer sure or almost sure):
2.) Q: Why does the DO-328 has a TOGA button although it has no autothrottle?
2.) A: When the TOGA Button has been pressed the Bugs on TRQ gauges will be set to TO Values. I will send you an flight test report, where this Feature has been mentioned.
3.) Q: What is shown on MFD page "MAIN 2/2"?
3.) A: There must be at least "MAINT" (=maintenance, see exceedance page, standard specification p. 31-23)
5.) Q: What is the function of the small lever on the left and right panel, beside the mainpanel?
5.) A: Should be Air conditioning, the texts are "AIRFLOW" and maybe "FEET" (pull the lever if you have cold feet)
6.) Q: What is the correct color (hex value) of the MFD/EICAS-lines (currently #00ffff in issue17, before #3282dc)?
6.) A: #55bbee

**Comment by xcvb85** [comment]
1.) There are two PFD styles (the one where the FD is shown as two lines and the one where the FD is shown as chevron). Where can I switch between them?
2.) Why does the DO-328 has a TOGA button although it has no autothrottle?
3.) What is shown on MFD page MAIN 2/2?
4.) What is the content of "REF DATA"->"CRUISE"?

**Comment by HHS81** [comment]
2.) When the TOGA Button has been pressed the Bugs on TRQ gauges will be set to TO Values. I will send you an flight test report,  where this Feature has been mentioned.

**Comment by ghost** [comment]
4 could maybe be vno and vne?



------------------------------------------------------------------------
Gesendet mit der Telekom Mail App
<http://www.t-online.de/service/redir/email_app_android_sendmail_footer.htm>


--- Original-Nachricht ---
Von: HHS81
Betreff: Re: [HHS81/do328] Questions (#49)
Datum: 04.01.2017, 20:22 Uhr
An: HHS81/do328
Cc: Subscribed



2.) When the TOGA Button has been pressed the Bugs on TRQ gauges will be 
set to TO Values. I will send you an flight test report, where this Feature 
has been mentioned.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/HHS81/do328/issues/49#issuecomment-270461504> , or 
mute the thread

**Comment by xcvb85** [comment]
> 4 could maybe be vno and vne?

All I could find was this picture:
http://cdn-www.airliners.net/photos/airliners/5/4/5/0208545.jpg

SK1: VC
SK2: ???
SK3: Could be VSTD
SK4: Could be L80% R80%

**Comment by HHS81** [comment]
SK4 is 80.7 and that's the prop rpm in percent which should be used at cruise. From a Video on youtube I can see that Pressing the soft buttons changes the flight state:TO, Climb, Cruise and Landing. TO and landing will show VSpeeds and 100% prop speed.

**Comment by HHS81** [comment]
Climb will show prop speed of 84, 6% and probably calculated recommended Climb speed by FMC.

**Comment by xcvb85** [comment]
Yes you are right. It's 80.7 and not 80%. I will change that. Next mistake: Speed marks in PFD are 1, R and 2 when in TO mode and not V1, VR and V2. I will also fix that. For some reason I only used REF for LANDG mode which is correct. CLIMB and CRUISE modes are not yet implemented.

**Comment by HHS81** [comment]
I just saw  that I gave the link to article I mentioned is in the readme.it is the article by flightglobal

**Comment by xcvb85** [comment]
1.) I can imagine that for security reasons this must be done by the mechanics on the ground and that  there is no way to change that by pressing a simple button. We could change the style depending on the selected livery but this is of course very low priority.

**Comment by HHS81** [comment]
4.) VMO and VNE?

**Comment by HHS81** [comment]
What is the function of  the small lever on the left and right panel, beside the mainpanel?:
[https://d31asmy75eposw.cloudfront.net/full/1/85489_1146152359.jpg](https://d31asmy75eposw.cloudfront.net/full/1/85489_1146152359.jpg)

**Comment by xcvb85** [comment]
Not sure which one you mean. The lever where only the half of it is shown in the very bottom left in this picture?
http://cdn-www.airliners.net/photos/airliners/0/3/9/0866930.jpg?v=v40
Maybe something like air conditioning?

**Comment by HHS81** [comment]
Yep,  that one.

**Comment by xcvb85** [comment]
Here you can see that the texts are "AIRFLOW" and maybe "FEET".
http://www.freundeskreis-dornier.de/joomla/images/Do%20328%20vor%20Restauration%201.jpg

**Comment by xcvb85** [comment]
6.) What is the correct color (hex value) of the MFD/EICAS-lines (currently #00ffff in issue17, before #3282dc)?

**Comment by xcvb85** [comment]
6.) I used #55bbee and I guess it is quite a good match. For hatchings (e.g. fuel, flight control) I used #3bb0eb which is #55bbee with reduced brightness.

**Comment by xcvb85** [comment]
~~1.) I guess this could be done with one of the knobs of the "Red Warning Panel and Reversionary Controller" as shown in specification 31-18. But not sure which one. Maybe "ATT/HDG"?~~

**Comment by HHS81** [comment]
How to control the Throttle/ Condition Levers in the cockpit?

Now, we have a knob animation from GI to Full Power. 
But the levers also include CUT OFF/ Reverse Thrust. 

The real thing has a latch on each Throttle lever, which has to be lifted, when reaching a certain point, so the throttle lever can be pulled back. When you lift the latch before this certain point or don't lift, you can't pull back the levers - no reverse thrust on the prop.

How we should control this from the cockpit?

**Comment by xcvb85** [comment]
Reverse Thrust:
I'm not sure if everything can be simulated in that high detail. The flap lever also has latches which you must pull before you can set the flaps and as far as I know you cannot prevent that you can set the flaps just by pressing "[" or "]". The del-key is the standard for reverse thrust in Flightgear even if every aircraft has some kind of protection against activating reversers in flight. I guess I would leave it as it is.

Cut Off:
I'm not sure how exactly Cut Off works here. Maybe there could be a point where you must do a mouse click?

**Comment by HHS81** [comment]
@xcvb85 
Do you have more information how the flap lever hatches looks like and work?

Reverse Thrust: The misuse of the throttle latches was cause of every rwy overrun by the Dornier 328, and there had been a lot. So I would like to simulate this in any way. 
In the sim  you can only enter Beta and reverse by pressing Del-key in the moment, but in any position of the throttle. I thought about a script which makes Del-key allowing to enter beta/Reverse only certain throttle positions. So you can't get reverse thrust when you have full throttle.

Cut Off:
Prop: Cold and dark, the Condition lever is at cut off position. When starting engines you pushes the condition levers forward into feather position, then into low 
Jet: Cold and dark, the Throttle levers ar at cut off position. When starting engines you pushes the lvers forward into idle.

Maybe the mouse click solution is the best here indeed

**Comment by xcvb85** [comment]
I got the information about the flaps lever from an Infowerk documentation. Sometimes they contain really good stuff but unfortunately they sometimes mix things with other planes. The listed mechanics cannot be the one from the Do-328. Sorry for the confusion.

**Comment by xcvb85** [comment]
3.) MFD PAGE 2/2 is shown in this video:
https://www.youtube.com/watch?v=DF1fzD5z5bY
but the quality is too bad to be able to see anything. At least you can hear that the first softkey is "TEST" and the content of the test menu.

**Comment by HHS81** [comment]
What's this panel? Text? Function?
![do328question](https://user-images.githubusercontent.com/12649375/35482514-ef81d6fc-0436-11e8-94be-0ca6ca2908b6.jpg)

**Comment by xcvb85** [comment]
> What's this panel? Text? Function?

This could be the radio for the flight observer (sitting on the jump seat). Maybe something like this: https://magazin.lufthansa.com/xx/en/fleet/embraer-195-en/panorama-photo-cockpit-embraer-erj-195/. Sorry, but I didn't find any better pictures either.
